main.go

```diff
@@ -25,12 +27,13 @@ AND summary ~ %q
 ORDER BY created DESC`
 
 func main() {
-
 	p := params{}
+	flag.StringVar(&p.csvOutput, "csv", "", "Convert XML to a CSV file (use dash [-] for stdout)")
```

Comment:
Maybe name this to `csv-output` instead? Maybe its more clear then that you have to specify `-`, first time reading this I'd expected `--csv` to indicate that I want csv as a format, if that makes sense.